Modules/ThirdParty/DCMTK/DCMTKGitTag.cmake

```diff
@@ -19,4 +19,4 @@ set(DCMTK_GIT_REPOSITORY "https://github.com/InsightSoftwareConsortium/DCMTK.git
 
 
 
-set(DCMTK_GIT_TAG "972d43e13acde08c186f6e02816b18d2fbaba4cc") # 20200214_DCMTK_PATCHES_FOR_ITK
+set(DCMTK_GIT_TAG "cba5794fb294ca7a473a70dd97fdee115922a348") # 20210117_DCMTK_PATCHES_FOR_ITK
```

Comment:
The name of the branch should be 20220117_DCMTK_PATCHES_FOR_ITK (start with 2022, not 2021).